charts/aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -80,7 +81,7 @@ spec:
             timeoutSeconds: 3
             periodSeconds: 10
             failureThreshold: 5
-          {{- with .Values.resources }}
+          {{- with .Values.node.resources }}
```

Comment:
I am not a big fan of this change. This would be a breaking change and force the operators to update their setup. Can we somehow give this new value priority and if it doesn't exist fallback to `.Values.resources`? We can add a deprecation note for the old field on the values file.

Reply:
Good catch ^^

Pushed some changes, adding a condition so the additions would be backwards compatible. Tell me if something like that would work for you :)